### ISSUE TITLE
Data exclusion for verify.Values

### ISSUE THREAD

**Issue by mikelangelon** [pascaldekloe/goe#2]
It would be super cool to have an exclude list for the verify function. Sometimes the match is gonna be between complex or deep structs and right now is mandatory to provide an exact struct match. Which is perfect plenty of times, but in some scenarios could be interesting to just care about specific values for the match.

Like, let's suppose that I'm expecting an struct like: 

> type PeriodCalc struct {
>   Something string
>   Blablabla string
>   Value string 
>   Trace []*Log
> }

But maybe I don't care about this Trace array and I want just to ignore it and check the other variables.

**Comment by pascaldekloe** [comment]
Makes sense indeed. Maybe a (GoEL) exclusion list as a vararg would do. I'll give it a shot later this week...
